src/DssSpell.sol

```diff
@@ -18,24 +18,31 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-
 import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
-import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
 
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+interface RwaLiquidationLike {
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface InputConduitJarLike {
+    function push(uint) external;
```

Comment:
I would suggest to use explicit type, just like in the [contract itself](https://etherscan.io/address/0x6934218d8b3e9ffcabee8cd80f4c1c4167afa638#code#F5#L238), i.e. `uint256`

Reply:
fixed in a69e26a